javascript/javascript1/week3/Javascript_warmup.js

```diff
@@ -0,0 +1,110 @@
+// #1 Item array removal
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmaad";
```

Comment:
There is a typo here which breaks the example. 😢 Good job on the implementation though! 👍